modules/bling/README.md

```diff
@@ -19,4 +19,22 @@ install:
     # - gnome-vrr # enables gnome-vrr for your image 
     # - container-tools # installs container-related tools onto /usr/bin: kind, kubectx, docker-compose and kubens 
     # - laptop # installs TLP and configures your system for laptop usage
+    # - flatpaksync # allows synchronization of user-installed flatpaks, see separate documentation section
 ```
+## Submodule documentation
+
+### `flatpaksync`
+
+The `flatpaksync` submodule can be used to synchronize user-installed flatpaks into a gist or traditional repository.
+
+Once the submodule is activated, users can create a file `$HOME/.config/flatpaksync/env` informing the repository that will be used to synchronize their apps in the POSIX standard:
+
+```bash
+GIST_REPO=<YOUR_REPO>
```

Comment:
Overall, this documentation leaves the purpose of this module unspecified. Does this module upload a list of flatpaks installed on your system to a Github Gist (a proprietary Github product), or is any `git` repository supported? There should be examples of what to set `GIST_REPO` to. Or, does the sync happen in two directions, where editing the file in the gist changes the flatpaks on your system as well? What is the structure of the repository? Is the repository cloned into `~/.config/flatpaksync/` or where exactly?

And a big question; What is the usecase where this approach has benefits over the `default-flatpaks` module? 

Reading the documentation multiple times and reading the source code as well does give me answer to these questions, which I had while reading the docs for the first time. It's already pretty good as of now, but needs a bit of polish. And if you're more comfortable with someone else writing the documentation, feel free to just explain stuff as a reply and I'll transform that into docs on a later date.

Reply:
It can be any git repository, actually. In my case, I used a gist because I found it simpler. Synchronization is bidirectional, every time the user installs a flatpak using the `--user` clause, the `flatpak.list `file will be updated. Maybe changing the variable name would help...

The repository is cloned to `/tmp/sync`. It is not necessary to set up any structure in the repository, the script will take care of creating the `flatpak.list` file in the root of the repository, if necessary, or updating it.

I didn't make this clear in the documentation because I wanted to make this timing transparent. Of course, any user with a little more experience will be able to understand everything I say by looking at the `flatpakcheckout` and `flatpaksync` scripts.

From what I understand from the `default-flatpaks` module, it will enable repositories and install apps, either in the system context or in the user context, but it will not synchronize the apps installed by the user over time. This submodule does not have the same function, using it the user can synchronize their flatpaks, which will not always be the same as the ones other PC users use, and facilitate their installation if they format the computer, for example.

Please feel free to rewrite the documentation as you see fit.

Reply:
Thanks for the quick and clear response! Maybe the variable should be named `GIT_REPO` then. I'm gone for today but I'll do some more changes as I see fit and merge the PR soon.